src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -9,37 +9,142 @@
+			},
+			{
+				"name": "x_forwarded_for",
+				"type" : "string",
+				"comment" : "The HTTP Header contains IP address of user connecting through a proxy. see: https://en.wikipedia.org/wiki/X-Forwarded-For"
+			},
+			{
+				"name": "via",
+				"type" : "string",
+				"comment" : "The HTTP Header Via informs the server of proxies through which the request was sent."
+			},
+			{
+				"name": "thread_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the thread in which the request was processed."
+			},
+			{
+				"name": "elapse_ms",
+				"type" : "bigint",
+				"comment" : "The total time consumed processing the user request in milliseconds."
+			},
+			{
+				"name": "success",
+				"type" : "boolean",
+				"comment" : "Indicates if the user request succeeded (true) or failed (false)."
+			},
+			{
+				"name": "stack",
+				"type" : "string",
+				"comment" : "The stack (prod, dev) on which request was made."
+			},
+			{
+				"name": "instance",
+				"type" : "string",
+				"comment" : "The version of the stack that processed the request."
+			},
+			{
+				"name": "vm_id",
+				"type" : "string",
+				"comment" : "The unique identifier of the actual EC2 instance that processed the request."
+			},
+			{
+				"name": "return_object_id",
+				"type" : "string",
+				"comment" : "The Synapse id of the object affected by the request, if available."
+			},
+			{
+				"name": "query_string",
+				"type" : "string",
+				"comment" : "The set of characters tacked onto the end of a URL after the the question mark (?)."
+			},
+			{
+				"name": "response_status",
+				"type" : "bigint",
+				"comment" : "The response code of request, e.g., 200, 401, 500 etc."
+			},
+			{
+				"name": "oauth_client_id",
+				"type" : "string",
+				"comment" : "The unique id of the oauth client used in the request. It will be empty in case request is not made by OAuth client."
+			},
+			{
+				"name": "basic_auth_username",
+				"type" : "string",
+				"comment" : "For requests that are authenticated with Basic Authentication, this is the content of the user-name field."
+			},
+			{
+				"name": "auth_method",
+				"type" : "string",
+				"comment" : "The authentication method used by the client. Currently BEARERTOKEN, SESSIONTOKEN, BASIC, APIKEY methods are supported."
+			},
+			{
+				"name": "normalized_method_signature",
+				"type" : "string",
+				"comment" : "The simplified version of the request url. It includes the http method and the path with all id's extracted."
+			},
+			{
+				"name": "client",
+				"type" : "string",
+				"comment" : "The is an alias of the user agent e.g., WEB, JAVA, PYTHON etc."
+			},
+			{
+				"name": "client_version",
+				"type" : "string",
+				"comment" : "The version of the client used to make the request."
+			},
+			{
+				"name": "entity_id",
```

Comment:
Whats the difference beteween this and "return_object_id"?

Reply:
return_object_id is the id returned to the user in the response body. entity_id is the id used by user in the request url.